Content.Server.Database/Model.cs

```diff
@@ -259,6 +266,21 @@ public class Round
         public List<Player> Players { get; set; } = default!;
 
         public List<AdminLog> AdminLogs { get; set; } = default!;
+
+        [ForeignKey("Server")] public int ServerId { get; set; }
+        public Server Server { get; set; } = default!;
+    }
+
+    [Table("server")]
+    public class Server
+    {
+        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+
+        public string Name { get; set; } = default!;
+
+        [InverseProperty(nameof(Round.Server))]
+        public List<Round> Rounds { get; set; } = default!;
```

Comment:
All these attributes you added are automatically inferred by EFCore, you can get rid of them.

Reply:
I know that this didn't work in ss14.admin when the Rounds didn't have the inverse property annotation so I'm very unkeen about removing that but I removed the table annotation.